scripts/sct_dmri_moco.py

```diff
@@ -99,6 +99,12 @@ def get_parser():
                       mandatory=False,
                       default_value=param_default.path_out,
                       example='dmri_moco_results/')
+    parser.add_option(name='-interleaved',
+                      type_value='multiple_choice',
+                      description='Interleaved acquisition: 0 = NOT-interleaved, 1 = interleaved',
+                      mandatory=False,
+                      default_value='0',
+                      example=['0','1'])
```

Comment:
i would use int here (i know the rest of the code uses str, but that was a mistake).

Reply:
It looks that currently used parser (`from msct_parser import Parser`) does not support int. When I tried int, I got error. When I briefly checked other sct scripts using this parser (`sct_dmri_transpose_bvecs.py` or `sct_dmri_create_noisemask.py`), all of them use str. For the future, we can consider to switch to `argparse`.

Reply:
indeed, we eventually need to get rid of the custom `msct_parser`.